examples/frameworkless-on-openshift-with-docker-example/readme.md

```diff
@@ -0,0 +1,88 @@
+# Frameworkless Rest API on OpenShift 
+
+
+The purpose of this example is to demonstrate the following:
+
+- How you can use the openshift-annotations to configure a web app and the services.
+- How you can end-to-end test the application.
+- How you can trigger an image build after the compilation.
+
+
+The application is using:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>openshift-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+Which contains the annotation processors that detect web applications.
+
+The [App.java](src/main/java/io/dekorate/example/fwlonopenshift/App.java) is annotated with `@OpenshiftApplication` which triggers the resource generation.
+
+```java
+@OpenshiftApplication(
+        name = "frameworkless-openshift",        
+        ports = @Port(name = "web", containerPort = 8080),  
```

Comment:
Same as with the kubernetes example. We need to align this (remove uneeded options)